internal/utils/validation.go

```diff
@@ -162,3 +165,26 @@ func ValidateNationalIDVerification(nationalID string) error {
 
 	return nil
 }
+
+// ValidateURLScheme checks if a URL is valid and if it has a valid scheme.
+func ValidateURLScheme(link string, scheme ...string) error {
+	// Use govalidator to check if it's a valid URL
+	if !govalidator.IsURL(link) {
+		return errors.New("invalid URL format")
+	}
+
+	// Parse the URL to enforce HTTPS
```

Comment:
I believe the comment above is misleading because it makes it sound like we're enforcing HTTPS in the next block, but we're not.